### PR TITLE
add-key option for generate key cli command

### PR DESCRIPTION
**Pull request by solarw**
## Proposed changes

add-key option for generate key cli command

## Fixes

https://github.com/fetchai/agents-aea/issues/2093

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [x] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [x] Lint and unit tests pass locally with my changes and CI passes too
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that code coverage does not decrease.
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by DavidMinarsch — APPROVED** [review]
LGTM

File: tests/test_cli/test_generate_key.py [hunk 1]
```diff
@@ -189,3 +191,48 @@ def teardown_class(cls):
...
+
+            result = self.run_cli_command(
+                "config", "get", self.keys_config_path, cwd=self._get_cwd()
+            )
+            assert result.exit_code == 0
+            agent_keys = json.loads(result.stdout_bytes)
+            assert agent_keys.get(FetchAICrypto.identifier) == FETCHAI_PRIVATE_KEY_FILE
+
+
+class TestGenerateKeyWithAddKeyWithConnection(AEATestCaseEmpty):
```
Comment (5A11) [inline tests/test_cli/test_generate_key.py:234]: Is this complete?

Comment (solarw) [reply tests/test_cli/test_generate_key.py:234]: i think so

Comment (5A11) [reply tests/test_cli/test_generate_key.py:234]: There is no test in this class.

Comment (5A11) [reply tests/test_cli/test_generate_key.py:234]: Maybe you meant to subclass `TestGenerateKeyWithAddKeyWithoutConnection`?

Comment (DavidMinarsch) [reply tests/test_cli/test_generate_key.py:234]: Yep

Comment (solarw) [reply tests/test_cli/test_generate_key.py:234]: sorry. missed this. added

Comment (5A11) [reply tests/test_cli/test_generate_key.py:234]: Still haven't :)

**Review by 5A11 — APPROVED** [review]
lgtm

**Pull request merged**